react-native-weather-sensor-module.podspec

```diff
@@ -0,0 +1,35 @@
+require "json"
+
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+folly_version = '2021.06.28.00-v2'
+folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
+
+Pod::Spec.new do |s|
+  s.name         = "react-native-weather-sensor-module"
+  s.version      = package["version"]
+  s.summary      = package["description"]
+  s.homepage     = package["homepage"]
+  s.license      = package["license"]
+  s.authors      = package["author"]
+
+  s.platforms    = { :ios => "10.0" }
+  s.source       = { :git => "https://github.com/spindance.code-challenge.mobile.react-native.weather-sensor-service-package/tree/develop.git", :tag => "#{s.version}" }
+
+  s.source_files = ['ios/', 'ios/spindance.code-challenge.mobile.ios.weather-sensor-service-package/Sources/WeatherSensorService/**/*.swift']
+  s.dependency "React-Core"
+
+  # Don't install the dependencies when we run `pod install` in the old architecture.
+  if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then
+    s.compiler_flags = folly_compiler_flags + " -DRCT_NEW_ARCH_ENABLED=1"
+    s.pod_target_xcconfig    = {
+        "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost\"",
+        "CLANG_CXX_LANGUAGE_STANDARD" => "c++17"
+    }
+
+    s.dependency "React-Codegen"
+    s.dependency "RCT-Folly", folly_version
+    s.dependency "RCTRequired"
+    s.dependency "RCTTypeSafety"
+    s.dependency "ReactCommon/turbomodule/core"
+  end
+end
```

Comment:
Just curious - why is there a podspec? How does Cocoapods fit into this? I'd think that would only come into play for iOS native.

Reply:
I am under the assumption that it is need to point to the ios repo. So when I add this library to another project it reads the podspec and knows where to look for additional ios code. It is needed for other repos to find the ios code.

Reply:
So maybe the RN Xcode project points at the Swift package via a cocoapod? Man that seems weird to me.

Reply:
Ya that sounds right. This was part of the auto generated code from [react-native-builder-bob](https://github.com/callstack/react-native-builder-bob). Not sure why they did it that way.